doc/getting_started.md

```diff
@@ -34,127 +35,69 @@ class AppKernel extends Kernel
 }
 ```
 
+or, if you are using [the new flex structure](https://symfony.com/doc/current/setup/flex.html):
+```php
+<?php
+// config/bundles.php
+
+return [
+    // …
+    Prooph\Bundle\ServiceBus\ProophServiceBusBundle::class => ['all' => true],
+];
+```
+
 ## Configure your first command bus
 
 There are three different types of message bus supported by the ProophServiceBus.
 While they have totally different purposes, their configuration is nearly the same.
 As an example, we will configure a command bus.
 For query bus and event bus, please have a look at the [configuration reference](./configuration_reference.html). 
 
-The command bus is configured in `app/config/config.yml`:
+The command bus is configured in `app/config/config.yml`
+(or `config/packages/prooph_service_bus.yml` if you are using flex):
```

Comment:
Learned from @chr-hertel and @xabbuh yesterday that Yaml config file extension is now `yaml` (official standard)
So we should change that.